servicemesh-install/topics/appendix-threescale-adapter.adoc

```diff
@@ -169,15 +170,23 @@ By using the refreshing process, cached values whose hosts become unreachable wi
 
 = Authenticating requests
 This Technology Preview release supports the following authentication methods:
+
 * *Standard API Keys*: single randomized strings or hashes acting as an identifier and a secret token.
 * *Application identifier and key pairs*: immutable identifier and mutable secret key strings.
```

Comment:
"OpenID authentication method" needs to be added here

Reply:
@vramosp and @dfennessy I've added text for "OpenID authentication method" per Vanessa's request. Please review.